### PR TITLE
Update docs for 4844

### PR DESCRIPTION
**Pull request by teddyknox**

### CODE REVIEW

File: docs/eigenda/operator-guides/overview.md [hunk 1]
```diff
@@ -16,14 +16,22 @@ and documentation.
...
-
-|                         | Supported Throughput | Max % of Stake      |
-| ----------------------- | -------------------- | ------------------- |
-| General Purpose - large | 80 Kbps              | 0.03% (Solo staker) |
-| General Purpose - xl    | 500 Kbps             | 0.2%                |
-| General Purpose - 4xl   | 50 Mbps              | 20%                 |
+The EigenDA network design dictates that operators with greater stake will
+receive more frequent chunk dispersal requests, since they operate with greater
+cryptoeconomic security. This means that an operator's node requirements are a
+function of the total amount of stake the wield across all quorums, which we
```
Comment (dabit3) [inline docs/eigenda/operator-guides/overview.md:22]: the -> they
"amount of stake the wield" -> "amount of stake they wield"

File: docs/eigenda/operator-guides/overview.md [hunk 2]
```diff
@@ -34,18 +42,19 @@ are introduced.
 
 ### Node Storage Requirements
 
-EigenDA nodes must provision storage capacity (high performance SSD volumes) in
-order to store and serve all data passed to the node over a 2-week period.
+EigenDA nodes **must** provision high-performance SSD storage in order to keep
+up with network storage and retrieval tasks. Failure to maintain adaquate
```
Comment (dabit3) [inline docs/eigenda/operator-guides/overview.md:46]: adaquate -> adequate

File: docs/eigenda/operator-guides/overview.md [hunk 3]
```diff
@@ -16,14 +16,22 @@ and documentation.
...
+call 'Total Quorum Stake' (TQS). For example, if an operator Foobar has 3% stake
+on the restaked ETH quorum, and 5% ETH on a staked WETH quorum, then operator
+Foobar's TQS is 8%.
+
+Operators should use the following table to determine which [EigenLayer node class][1]
+is appropriate for their level of stake:
+
+| Total Quorum Stake (TQS) |                         | Supported Throughput |
+| ------------------------ | ----------------------- | -------------------- |
+| 0.03% (Solo staker)      | General Purpose - large | 80 Kbps              |
```
Comment (mooselumph) [inline docs/eigenda/operator-guides/overview.md:32]: Maybe these should say "Up to 0.03%, Up to 0.2%, Up to 20%"

File: docs/eigenda/operator-guides/overview.md [hunk 4]
```diff
@@ -34,18 +42,19 @@ are introduced.
 
 ### Node Storage Requirements
 
-EigenDA nodes must provision storage capacity (high performance SSD volumes) in
-order to store and serve all data passed to the node over a 2-week period.
+EigenDA nodes **must** provision high-performance SSD storage in order to keep
+up with network storage and retrieval tasks. Failure to maintain adaquate
+performance will result in automatic churn.
```
Comment (mooselumph) [inline docs/eigenda/operator-guides/overview.md:47]: `will result in unacceptable validation latency and automatic churn.`?

File: docs/eigenda/operator-guides/overview.md (hunk 2 above)
Comment (teddyknox) [reply docs/eigenda/operator-guides/overview.md:46]: good catch

**Pull request merged**